jenkins/runner

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This script exists to ensure that `bash` is the process 1 in the container
+# executed by jenkins/run.  If `bash` is not process 1, then any zombie
+# processes won't be reaped until the program running as process 1 explicitly
+# reaps zombie processes.
+
+if [[ ${#} -ne 1 ]]; then
+    printf -- "${0}: Unexpected number of arguments: ${#}\n" >&2
+    exit 1
+fi
+eval "${1}"
```

Comment:
Why is this script restricted to a single-element command?  (Why not just `eval "${*}"` or the equivalent?)  This greatly limits what we can do with `jenkins/run`!

With this change, I think I can still do things like `jenkins/run bash`, but I won't be able to do things like `jenkins/run make srpm`.  😢

Reply:
Because it is married to `jenkins/run` which passes all its arguments as 1 argument.

Reply:
So, the intention is that the entire command line will be passed as a single (quoted) argument, this script counts on that, and so it performs the check (which, under those circumstances, is a Good Thing).

However, wouldn't it work just as well to use `"${*}"` in the `eval` statement?  Then we wouldn't need to do the check, and there would be no confusion.

Reply:
I think I have a way to make this a bit better.  Will post shortly.